src-docs/src/components/guide_section/guide_section.tsx

```diff
@@ -135,7 +135,7 @@ export const GuideSection: FunctionComponent<GuideSection> = ({
     const isPlaygroundUnsupported =
       typeof window !== 'undefined' &&
       typeof document !== 'undefined' &&
-      !!window.MSInputMethodContext &&
+      !!(window as any).MSInputMethodContext &&
```

Comment:
Maybe, we could consider completely removing this. Please look at the follow up issue #886

Reply:
Good point about fixing it by adding to global types. The usage of `any` is common to highlight areas of uncertainty or complexity in typescript. It is usually recommended to avoid it if possible

In this case it should be safe to mark some pain point by `as any`. It could be more scary to cleanup `index.d.ts` in future, than cleanup local usage of `any`. My preference is to keep the`index.d.ts` nice and clean. 

On other hand I cant see better cast here and we have issue #886 for refactoring